package.json

```diff
@@ -1,29 +1,75 @@
 {
   "name": "@storybook/root",
+  "private": true,
+  "type": "module",
+  "workspaces": {
+    "packages": [
+      "code",
+      "code/core",
+      "code/addons/*",
+      "code/frameworks/*",
+      "code/lib/*",
+      "code/builders/*",
+      "code/presets/*",
+      "code/renderers/*",
+      "scripts"
+    ]
+  },
   "scripts": {
     "ci-tests": "cd code; yarn ci-tests",
     "get-report-message": "cd scripts; yarn get-report-message",
     "get-sandbox-dir": "cd scripts; yarn get-sandbox-dir",
     "get-template": "cd scripts; yarn get-template",
-    "i": "yarn --cwd scripts && yarn --cwd code",
+    "i": "yarn",
     "knip": "cd code; yarn knip",
     "lint": "cd code; yarn lint",
-    "nx": "cd code; yarn nx",
+    "nx": "nx",
     "pretty-docs": "cd scripts; yarn install >/dev/null; yarn docs:prettier:write",
     "start": "yarn task --task dev --template react-vite/default-ts --start-from=install",
-    "svelte-ecosystem-ci:before-test": "node ./scripts/ecosystem-ci/before-test.js svelte-kit/skeleton-ts && cd ./sandbox/svelte-kit-skeleton-ts && yarn install",
+    "svelte-ecosystem-ci:before-test": "node ./scripts/ecosystem-ci/before-test.js svelte-kit/skeleton-ts && cd ../storybook-sandboxes/svelte-kit-skeleton-ts && yarn install",
     "svelte-ecosystem-ci:build": "yarn task --task install && yarn --cwd code build svelte && yarn task --task sandbox --template svelte-kit/skeleton-ts --start-from=compile --no-link --skip-cache",
     "svelte-ecosystem-ci:test": "yarn task --task test-runner --template svelte-kit/skeleton-ts --start-from=build --no-link --skip-cache && yarn task --task vitest-integration --template svelte-kit/skeleton-ts --start-from vitest-integration --no-link --skip-cache",
-    "task": "echo 'Installing Script Dependencies...'; cd scripts; yarn install >/dev/null; cd ..; yarn --cwd=./scripts task",
+    "task": "yarn --cwd=./scripts task",
     "test": "cd code; yarn test",
     "test:watch": "cd code; yarn test:watch",
     "upload-bench": "cd scripts; yarn upload-bench",
-    "vite-ecosystem-ci:before-test": "node ./scripts/ecosystem-ci/before-test.js react-vite/default-ts && cd ./sandbox/react-vite-default-ts && yarn install",
+    "vite-ecosystem-ci:before-test": "node ./scripts/ecosystem-ci/before-test.js react-vite/default-ts && cd ../storybook-sandboxes/react-vite-default-ts && yarn install",
```

Comment:
Please test all `ecosystem` scripts for Vite and Svelte manually to make sure they are not broken, if not already done.